src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -832,4 +838,166 @@ public void cannotCreateARequestWithUnknownPickupLocationServicePoint()
       hasMessage("Pickup service point does not exist"),
       hasUUIDParameter("pickupServicePointId", pickupServicePointId))));
   }
+
+  @Test
+  public void CanCreatePagedRequestWhenItemStatusIsAvailable()
```

Comment:
This appears to overlap with another test `canCreateAPageRequestForAvailableItem` in the same class, please can these be combined.

Reply:
`canCreateAPageRequestForAvailableItem` was created long before, but I removed it because it doesn't have explicit assertions that CanCreatePagedRequestWhenItemStatusIsAvailable() has.